doc/Type/Callable.pod6

```diff
@@ -30,7 +30,8 @@ of such a container is C<Callable>.
 
     method CALL-ME(Callable:D $self: |arguments)
 
-This method is required for postfix:«( )» and postfix:«.( )». It's what makes
+This method is required for L«C<( )> postcircumfix operator|/language/operators#postcircumfix_(_)»
```

Comment:
As long as we're modifying this, maybe add "the"?

Reply:
Sure ! Done.